recipe/meta.yaml

```diff
@@ -1,35 +1,45 @@
 {% set name = "solvation_analysis" %}
-{% set version = "0.1.5" %}
+{% set version = "0.3.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/solvation_analysis-{{ version }}.tar.gz
-  sha256: f33b867f42aace0d4f9199da86b21d2bd04a3cf9dab8ba039e0ee21bdd81b919
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/solvation-analysis-{{ version }}.tar.gz
+  sha256: 8dbf81b973e5e02c8dcd8e9247a794dd50bda701f4ce05fa681cd67e0bdb0ef8
 
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   number: 0
 
 requirements:
+  build:
+    - versioneer-518
   host:
-    - python >=3.7
+    - python >=3.8
     - pip
+    - pathlib
```

Comment:
I'll be honest, I'm very confused as to why you even have pathlib present. It's a core path of the Python library, you shouldn't need it externally. You are aware that by defining it in the pyproject.toml you're actually picking up the **old** pathlib from the Python 3.2 days right?

Reply:
I was not aware! This should definitely be removed.